docs/operations/hotplug_interfaces.md

```diff
@@ -79,27 +79,86 @@ Please refer to the
 for more info.
 
 Once the virtual machine is running, and the attachment configuration
-provisioned, the user can request the interface hotplug operation. Please refer
-to the following snippet:
-```bash
-virtctl addinterface vm-fedora --network-attachment-definition-name new-fancy-net --name dyniface1
+provisioned, the user can request the interface hotplug operation by 
+editing the VM spec template and add the desired interface and network:
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: vm-fedora
+template:
+  spec:
+    domain:
+      devices:
+        interfaces:
+        - name: defaultnetwork
+          masquerade: {}
+          # new interface
+        - name: new-fancy-net
+          bridge: {}
+    networks:
+    - name: defaultnetwork
+      pod: {}
+      # new network
+    - name: fancy-net
+      multus:
+        networkName: new-fancy-net
+ ...
 ```
-This will add the interface and network to the VM spec template and to the running VMI object.
 
-**NOTE**: You can use the `--help` parameter for more information on each
-parameter.
+The interface and network will be removed from the corresponding VMI object as well by Kubevirt controller.
+
+Alternatively, use JSON patch to automating the process: 
```

Comment:
I don't think the json patch example is required. It is just another way to update a spec.